test/integration/caa_test.go

```diff
@@ -28,7 +28,7 @@ func TestCAALogChecker(t *testing.T) {
 	stdErr := new(bytes.Buffer)
 	cmd.Stderr = stdErr
 	out, err := cmd.Output()
-	test.AssertEquals(t, string(out), "")
+	test.AssertContains(t, string(out), "Versions: ")
```

Comment:
Ideally this should assert that the output contains _only_ the "Versions:" line.